WordPress/WordPressTest/ReaderSiteServiceTests.swift

```diff
@@ -68,9 +64,137 @@ class ReaderSiteServiceTests: CoreDataTestCase {
             HTTPStubsResponse(jsonObject: [String: Any](), statusCode: 200, headers: nil)
         }
 
+        let service = makeService()
         let success = expectation(description: "The success block should be called")
-        self.service.unfollowSite(withID: 42, success: success.fulfill, failure: nil)
+        service.unfollowSite(withID: 42, success: success.fulfill, failure: nil)
         wait(for: [success], timeout: 0.5)
     }
 
+    func testFlagAsBlockedSuccessPath() {
+        let service = makeService()
+        let siteID: NSNumber = 42
+
+        stub(condition: isPath("/rest/v1.1/me/block/sites/\(siteID)/new")) { _ in
+            HTTPStubsResponse(jsonObject: ["success": 1], statusCode: 200, headers: nil)
+        }
+
+        waitUntil { done in
+            service.flagSite(
+                withID: siteID,
+                asBlocked: true,
+                success: {
+                    done()
+                },
```

Comment:
Nitpick: I think we could pass `done` as the `success` argument here?
```suggestion
                success: done,
```